src/nvim/ui.c

```diff
@@ -156,15 +167,18 @@ void ui_refresh(void)
   }
 
   int width = INT_MAX, height = INT_MAX;
+  bool pum_external = true;
 
   for (size_t i = 0; i < ui_count; i++) {
     UI *ui = uis[i];
     width = ui->width < width ? ui->width : width;
     height = ui->height < height ? ui->height : height;
+    pum_external &= ui->pum_external;
```

Comment:
well `&&=` doesn't exist, and as `bool` is semantically 1-bit it works correctly. But if the style guide or otherwise forbids it I will change it.